pyproject.toml

```diff
@@ -72,6 +72,9 @@ pytest-mock = ">=3.0"
 pykka = ">=3.0"
 tox = ">=3.24"
 
+[tool.poetry.scripts]
+install-daqmx-driver = 'nidaqmx._install_daqmx:install_daqmx_software'
```

Comment:
We should probably add a step [here ](https://github.com/ni/nidaqmx-python/blob/a7a3643911fd549cac198c0a2cc8dadd4a8ad8f4/.github/workflows/CI.yml#L14) that runs this script to validate that it continues to work.

Reply:
That would install DAQmx. Should the script have a command line option that causes it to exit without installing DAQmx (like --version)?

Reply:
or --help

Reply:
Added the --help option that will invoke the script but it will not install DAQmx driver